services/director-v2/tests/unit/test_modules_dask_client.py

```diff
@@ -112,76 +140,109 @@ def test_fct_add(x: int, y: int) -> int:
 )
 async def test_send_computation_task(
     dask_client: DaskClient,
+    user_id: PositiveInt,
+    project_id: ProjectID,
+    node_id: NodeID,
+    cluster_id: NonNegativeInt,
+    cluster_id_resource: str,
     image: Image,
     exp_annotations: Dict[str, Any],
     mocker: MockerFixture,
 ):
-    @retry(
-        stop=stop_after_delay(10),
-        wait=wait_random(0, 1),
-        retry=retry_if_exception_type(AssertionError),
-        reraise=True,
+    # INIT
+    exp_annotations["resources"].update(
```

Comment:
exp = exponential? experiment? exposition? expert? 🤪 ... ahhh expected 🤣

Reply:
I did it especially for you...